deploy/kubernetes/fluentd-sumologic.yaml.tmpl

```diff
@@ -727,3 +727,13 @@ spec:
 ---
 # Source: sumologic/templates/hpa.yaml
 
+---
```

Comment:
We will probably need to modify the CI process, such that the YAML for Tracing is rendered seperately. Otherwise, customers will pick it up in the non-helm installation.

Reply:
Does this action block merging?

Reply:
It looks like the YAML is only rendered when enabled is set to true, but would be good to confirm that is the behavior of the automation.  @rvmiller89 @vsinghal13 @samjsong can you confirm? If that is the case, it should be fine to merge.  we just want to make sure non-helm installs are affected.